evap/rewards/tools.py

```diff
@@ -2,11 +2,15 @@
 
 from django.conf import settings
 from django.contrib import messages
+from django.contrib.auth import user_logged_in
 from django.db import transaction
-from django.utils.translation import ugettext as _
+from django.utils import translation
+from django.utils.translation import ugettext as _, LANGUAGE_SESSION_KEY, get_language
```

Comment:
i suggest you either access `LANGUAGE_SESSION_KEY, get_language` via `translation.` or import `activate` here as well and remove `from django.utils import translation`. right now it's two different import styles for no apparent reason (apart from `_`, which is fine)